### PR TITLE
Demo updates

### PR DESCRIPTION
**Pull request by XhmikosR**
@feimosi: I don't remember your comments from the other time and forgot I had these changes, so please let me know if you want any changes.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -74,19 +74,27 @@
...
+.hljs-comment,.hljs-quote{color:#969896}.hljs-deletion,.hljs-name,.hljs-regexp,.hljs-selector-class,.hljs-selector-id,.hljs-tag,.hljs-template-variable,.hljs-variable{color:#c66}.hljs-built_in,.hljs-builtin-name,.hljs-link,.hljs-literal,.hljs-meta,.hljs-number,.hljs-params,.hljs-type{color:#de935f}.hljs-attribute{color:#f0c674}.hljs-addition,.hljs-bullet,.hljs-string,.hljs-symbol{color:#b5bd68}.hljs-section,.hljs-title{color:#81a2be}.hljs-keyword,.hljs-selector-tag{color:#b294bb}.hljs{display:block;overflow-x:auto;background:#1d1f21;color:#c5c8c6;padding:.8em}.hljs-emphasis{font-style:italic}.hljs-strong{font-weight:700}
```
Comment (feimosi) [inline demo/index.html]: Could you please make sure it's still the latest version?

File: demo/index.html [hunk 2]
```diff
@@ -74,19 +74,27 @@
...
     </style>
 </head>
 <body>
-<a href="https://github.com/feimosi/baguetteBox.js" class="sm-hidden">
-    <img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png">
+
```
Comment (feimosi) [inline demo/index.html]: That's an unnecessary empty line.

File: demo/index.html [hunk 3]
```diff
@@ -74,19 +74,27 @@
...
+<a href="https://github.com/feimosi/baguetteBox.js" class="ribbon sm-hidden">
+    <img src="img/forkme_right_darkblue_121621.png" alt="Fork me on GitHub" width="149" height="149">
```
Comment (feimosi) [inline demo/index.html]: Actually in this case `width` and `height` are not needed.

File: demo/index.html [hunk 4]
```diff
@@ -98,14 +106,30 @@
...
+            <img src="img/thumbs/1-5.jpg" alt="Image 1.5">
+        </a>
+        <a href="img/1-6.jpg">
+            <img src="img/thumbs/1-6.jpg" alt="Image 1.6">
+        </a>
+        <a href="img/1-7.jpg">
+            <img src="img/thumbs/1-7.jpg" alt="Image 1.7">
+        </a>
+        <a href="img/1-8.jpg">
+            <img src="img/thumbs/1-8.jpg" alt="Image 1.8">
```
Comment (feimosi) [inline demo/index.html]: Some time ago I read that meaningless `alt` attributes should be left empty:
https://davidwalsh.name/accessibility-tip-empty-alt-attributes
So I think you can remove that `Image 1.X` content with this PR.

File: demo/index.html [hunk 5]
```diff
@@ -162,18 +210,20 @@
...
     <a href="https://github.com/feimosi/baguetteBox.js">GitHub</a></h3>
 </div>
 <footer>
-    <p>Images have been downloaded from: <a href="http://unsplash.com/">unsplash.com</a> and <a href="http://www.gratisography.com/">gratisography.com</a></p>
+    <p>Images have been downloaded from: <a href="https://unsplash.com/">unsplash.com</a> and <a href="http://www.gratisography.com/">gratisography.com</a></p>
     <p>Copyright &copy; 2016 <a href="https://github.com/feimosi/">feimosi</a></p>
 </footer>
 <script src="js/baguetteBox.js" async></script>
-<script src="js/plugins.js" async></script>
+<script src="js/highlight.pack.js" async></script>
```
Comment (feimosi) [inline demo/index.html]: Change `pack` -> `min`

**Comment by XhmikosR** [comment]
All right, I can't spend any more time on this so closing.

**Comment by feimosi** [comment]
No need to get enraged. These were just small fixes. I even could have handle them myself if you didn't remove the PR branch...

**Comment by XhmikosR** [comment]
No hard feelings. It's just that I disagree with some of your suggestions
plus this branch was very old that I lost interest.

**Comment by feimosi** [comment]
Okay, I'm gonna reproduce the changes myself.

**Pull request closed without merging**